checker/tests/nullness/Issue738.java

```diff
@@ -0,0 +1,41 @@
+import org.checkerframework.checker.nullness.qual.NonNull;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+// Testcase for #738
+// https://github.com/typetools/checker-framework/issues/738
+// Also, see framework/tests/all-systems/Issue738.java
+public class Issue738 {
+    public static void methodA(int[] is, Object @Nullable [] os, int i) {
```

Comment:
I would remove the visibility modifiers and `static` from all the methods, to reduce noise.